src/containers/Downloads/downloadFilesData.js

```diff
@@ -63,15 +63,10 @@ function estimateGeneExpressionSize(dataSet) {
   let totalSize = 0;
 
   // calculate the size of each one of the experiments
-  for (let experimentId of Object.keys(dataSet)) {
+  for (let experimentId in dataSet) {
```

Comment:
I can see why the original version used `for/of` iteration because it is safer. If you are sure that dataSet is never an object that inherits from other object, then I guess `for/in` loop is okay.